client/extensions/woocommerce/app/dashboard/store-location-setup-view.js

```diff
@@ -215,11 +228,13 @@ class StoreLocationSetupView extends Component {
 					className="dashboard__pre-setup-address"
 					isEditable
 					onChange={ this.onChange }
+					showAllLocations
 				/>
 				<SetupFooter
+					busy={ this.state.isSaving }
 					disabled={ submitDisabled }
 					onClick={ this.onNext }
-					label={ translate( "Let's Go!" ) }
+					label={ translate( 'Next', { context: 'Label for button that submits a form' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **86** times:
`translate( 'Next', { context: 'Button label for a next revision'} )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/20877-alt)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?